docs/labels.md

```diff
@@ -6,88 +6,106 @@ weight: 10
-propagated from the referenced `Task` (if one exists, see the
-[Specifying a `Task`](taskruns.md#specifying-a-task) section of the `TaskRun`
-documentation) to the corresponding `TaskRun` and then to the `Pod`.
-
-For `Conditions`, labels are propagated automatically to the corresponding `TaskRuns`
-and then to `Pods`.
-
-## Automatically Added Labels
-
-The following labels are added to resources automatically:
-
-- `tekton.dev/pipeline` is added to `PipelineRuns` (and propagated to `TaskRuns`
-  and `Pods`), and contains the name of the `Pipeline` that the `PipelineRun`
-  references.
-- `tekton.dev/pipelineRun` is added to `TaskRuns` (and propagated to `TaskRuns`
-  and `Pods`) that are created automatically during the execution of a
-  `PipelineRun`, and contains the name of the `PipelineRun` that triggered the
-  creation of the `TaskRun`.
-- `tekton.dev/task` is added to `TaskRuns` (and propagated to `Pods`) that
-  reference an existing `Task` (see the
-  [Specifying a `Task`](taskruns.md#specifying-a-task) section of the `TaskRun`
-  documentation), and contains the name of the `Task` that the `TaskRun`
-  references.
-- `tekton.dev/clusterTask` is added to `TaskRuns` (and propagated to `Pods`) that
-  reference an existing `ClusterTask`and contains the name of the `ClusterTask` 
-  that the `TaskRun` references. For backwards compatibility, `TaskRuns` that
-  reference a `ClusterTask` will also receive `tekton.dev/task`.
-- `tekton.dev/taskRun` is added to `Pods`, and contains the name of the
-  `TaskRun` that created the `Pod`.
-- `app.kubernetes.io/instance` and `app.kubernetes.io/component` is added to 
-  Affinity Assistant `StatefulSets` and `Pods`. These are used for Pod Affinity for TaskRuns.
-
-## Examples
-
-- [Finding Pods for a Specific PipelineRun](#finding-pods-for-a-specific-pipelinerun)
-- [Finding TaskRuns for a Specific Task](#finding-taskruns-for-a-specific-task)
-
-### Finding Pods for a Specific PipelineRun
-
-To find all `Pods` created by a `PipelineRun` named test-pipelinerun, you could
-use the following command:
+## Label propagation
+
+Labels propagate among Tekton entities as follows:
+
+- For `Pipelines` instantiated using a `PipelineRun`, labels propagate
+automatically from `Pipelines` to `PipelineRuns` to `TaskRuns`, and then to
+the associated `Pods`.
+
+- Labels from `Tasks` referenced by `TaskRuns` within a `PipelineRun` propagate to the corresponding `TaskRuns`,
+and then to the associted `Pods`.
+
+- For standalone `TaskRuns` (that is, ones not executing as part of a `Pipeline`), labels
+propagate from the [referenced `Task`](taskruns.md#specifying-the-target-task), if one exists, to
+the corresponding `TaskRun`, and then to the associated `Pod`.
+
+- For `Conditions`, labels propagate to the corresponding `TaskRuns`, and then to the associated `Pods`.
+
+## Automatic labeling
+
+Tekton automatically addes the following labels to Tekton entities:
```

Comment:
Maybe it's worth adding a general statement that labels prefixed with "*.tekton.dev should be considered reserved for use by tekton, and should not be used by users.

Reply:
Added a note that clarifies this.